docs/pages/desktop-access/getting-started.mdx

```diff
@@ -295,7 +329,6 @@ windows_desktop_service:
     addr: "$LDAP_SERVER_ADDRESS"
     domain: "$LDAP_DOMAIN_NAME"
     username: '$LDAP_USERNAME'
-    password_file: /var/lib/teleport-svc-pass.txt
     # This should be the path to the certificate exported in Step 4.
```

Comment:
Where do we typically document feature deprecation? I would like to mention somewhere in our documentation that `password_file` is being deprecated in favor of a superior implementation.

Reply:
This seems like a good thing to flag in the release notes.

Reply:
Agreed, that makes sense.